### ISSUE TITLE
Role=status property is missing for Status related field in Publish screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#873]
**Describe the bug**
role=status property is missing for all status related fields.

**To Reproduce**

1. Go to Publish  (http://localhost:9992/cm/app/?view=publish&site=test10&path=%2FSites%2Ftest10%2F&pathType=page)
2. Click on a site. And then click on publish (Incremental or full).
3. The publishing status will be displayed under "Current status" heading.
4. Inspect elements given under status and you will find that role=status is missing for tr and td elements. 

**Expected behavior**
role=status property must be available for all status related fields.

**Additional context**
Navigate all the options available for similar issue and fix.
